cli/common/Cargo.toml

```diff
@@ -7,20 +7,20 @@ name = "solido-cli-common"
 version = "1.3.2"
 
 [dependencies]
-anchor-lang = "0.13.2"
+anchor-lang = "0.13.0"
```

Comment:
Why .0? Couldn’t we patch 0.13.2? Or is the fork based off of 0.13.2?

Reply:
Patching 0.13.0 and 0.13.2 is a twice more work than just patching 0.13.0

Reply:
0.13.0 is used in multisig, so I took a lower bound